pygmt/helpers/utils.py

```diff
@@ -306,5 +306,21 @@ def args_in_kwargs(args, kwargs):
     --------
     bool
         If one of the required arguments is in ``kwargs``.
+
+    Examples
+    --------
+
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"C": "xyz"})
+    False
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": "af"})
+    True
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": None})
+    False
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": True})
+    True
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": False})
+    False
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": 0})
+    True
     """
-    return any(arg in kwargs for arg in args)
+    return any(kwargs.get(arg) for arg in args)
```

Comment:
Need to modify this one-liner return statement so that `args_in_kwargs(args=["A", "B"], kwargs={"B": 0})` returns True.

Reply:
So, we should change it to:
```
return any(kwargs.get(arg) is not None for arg in args)
```
right?

Reply:
Close. I had to do `return any(kwargs.get(arg) is not None and kwargs.get(arg) is not False for arg in args)` so that arguments that are `False` don't count. Done in 7c3c6b4b0002b671df9c17ac21fd02272c05e8de